### ISSUE TITLE
Stopped working 

### ISSUE THREAD

**Issue by pattyland** [pyatyispyatil/github-releases-notify-bot#13]
Since a few weeks this but I had stopped sending me notifications and it seem to ignore all my comments. Is this projects that maintained?

**Comment by pyatyispyatil** [comment]
Yep, this project is still maintained, but I don’t have time to look for an error in db reading and fix it. Perhaps in the near future I will pull myself together and try to fix it.

**Comment by aaabramov** [comment]
Confirming not is working at the moment
